docs/UserGuide.md

```diff
@@ -134,57 +184,87 @@ Format: `edit INDEX [c/COMPANY] [r/ROLE] [s/STAGE] [d/DATETIME] [e/COMPANY_EMAIL
 
-### 4.3. Listing all existing internship applications: 
+### 6.3. Listing all existing internship applications: 
+
+Lists all of your internship applications in WorkBook in a [sorted order](#3-things-to-note).
 
-Lists all of your internship applications in WorkBook.
+<div markdown="block" class="alert alert-success">
+**:bulb: Tip:** Your most recent upcoming internship application will be at the top of the list!
+</div>
 
 Format: `list`
 
-### 4.4. Finding your internships by company, stage or role name:
+### 6.4. Finding your internships by company, stage or role name:
 
-Finds internships whose company, role or stage names contain any of the respective keywords.
+If you wanted to view all your internship applications corresponding to a particular company, stage or role,
+then this command is for you! <br>
+
+It can find internships whose company, role or stage names contain all the respective keywords.
 
 Format: `find c/COMPANY | r/ROLE | s/STAGE`
 
 * Displays a list of internships that match all the keywords.
+
+<div markdown="block" class="alert alert-success">
+**:bulb: Tip:** <br>
 * Case does not matter e.g. `meta` will match `Meta`.
-* Only full words will be matched e.g. `met` will not match `Meta`.
 * Order of the keywords does not matter e.g. `Jane Street` will match `Street Jane`.
+</div>
+
+<div markdown="block" class="alert alert-warning">
+**:exclamation: Caution**
+* Only full words will be matched e.g. `met` will not match `Meta`.
 * Exactly one attribute can be searched for either Company, Role or Stage.
+</div>
 
 
 Examples:
 * `find r/Engineer` returns `Software Engineer` and `Backend Engineer`.
 * `find s/Interview` returns `Technical Interview` and `Behavioural Interview`.
 
-### 4.5. Deleting your internship application:
+### 6.5. Deleting your internship application:
 
-Deletes the specified internship application from WorkBook.
+If you wanted to remove an internship application then this command
+deletes the specified internship application from WorkBook.
 
 Format: `delete INDEX`
 
 * Deletes the internship application at the specified `INDEX`.
 * The index refers to the index number shown in the displayed internship application list.
 * The index **must be a positive integer** 1, 2, 3, …​
 
+<div markdown="block" class="alert alert-warning">
+**:exclamation: Caution**
+* The command will only be executed if the index provided corresponds to an internship application
+</div>
+
 Examples:
 * `list` followed by `delete 2` deletes the 2nd internship application in WorkBook.
 * `find Meta` followed by `delete 1` deletes the 1st internship application within the results of the `find` command.
 
-### 4.6. Clearing your existing internship applications:
+### 6.6. Clearing your existing internship applications:
 
-Clears all internship applications in the WorkBook.
+It's a new cycle of summer internship applications and that means it's time for some spring-cleaning! 
```

Comment:
Gotta consider winter too, and there are many other potential uses of `clear`, so I was thinking:

Be it clearing your WorkBook for a new cycle of internships, or to get rid of our sample data, this command allows you to start with a clean WorkBook, removing all previously saved internship applications.

Then you can follow up with a tip box saying that all data in json files are immediately cleared after this command, so undo if you made a mistake and they will be back again or smth along those lines